contrib/platform/src/com/sun/jna/platform/win32/Kernel32.java

```diff
@@ -32,6 +31,32 @@
             Kernel32.class, W32APIOptions.DEFAULT_OPTIONS);
 
     /**
+     * <strong>LOAD_LIBRARY_AS_DATAFILE</strong> <br>
```

Comment:
What's up with all this HTML in the function descriptions. Looks like this was being cleaned up in other functions.